x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -484,13 +484,20 @@ export const EDIT_RULE_SETTINGS = i18n.translate(
   }
 );
 
-export const EDIT_RULE_SETTINGS_TOOLTIP = i18n.translate(
-  'xpack.securitySolution.detectionEngine.rules.allRules.actions.editRuleSettingsToolTip',
+export const LACK_OF_KIBANA_PRIVILEGES = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.actions.lackOfKibanaPrivileges',
   {
     defaultMessage: 'You do not have Kibana Actions privileges',
   }
 );
 
+export const LACK_OF_RULE_EDITING_PRIVILEGES = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.actions.lackOfRuleEditingPrivileges',
+  {
+    defaultMessage: 'You do not have rule editing privileges',
```

Comment:
It would be better to rephrase this line like "You do not have Kibana Security privileges" for consistency with the documentation and the Kibana feature privileges UI.